packages/engine/src/simulation/package/output/packages/json_state/mod.rs

```diff
@@ -76,12 +73,19 @@ impl GetWorkerSimStartMsg for JsonState {
 impl Package for JsonState {
     async fn run(&mut self, state: Arc<State>, _context: Arc<Context>) -> Result<Output> {
         let state = state.read()?;
-        let agent_states: std::result::Result<Vec<_>, crate::datastore::error::Error> = state
+        // Output packages can't reload state batches, since they only
```

Comment:
I agree with the contents of this comment, but why is it here in the run method of JsonState? I'm confused

We reload in `run_output` I don't think packages should have to think about reloading at all?

Reply:
Let's move the comment into `run_output`?

Reply:
Moved it